lib/model/QubitInformationObject.php

```diff
@@ -440,6 +440,11 @@ public static function getUpdatedRecords($options = array())
       $criteria = QubitAcl::addFilterDraftsCriteria($criteria);
     }
 
+    if (isset($options['topLevel']) && $options['topLevel'])
```

Comment:
How do you feel about
`if (!empty($options['topLevel']))`

It's still a fairly loose check but we do it all over the application already and it's more condensed.